cmd/os-readdir_unix.go

```diff
@@ -84,11 +84,14 @@ func readDir(dirPath string) (entries []string, err error) {
 	return readDirN(dirPath, -1)
 }
 
-// readDir applies the filter function on each entries at dirPath, doesn't recurse into
+// readDir applies the fn() function on each entries at dirPath, doesn't recurse into
```

Comment:
Could you update the comment of this function and say we return nil instead of errFileNotFound and why ?

Reply:
Sure